src/config/config.cc

```diff
@@ -159,6 +159,7 @@ Config::Config() {
       {"migrate-sequence-gap", false, new IntField(&sequence_gap, 10000, 1, INT_MAX)},
       {"unixsocket", true, new StringField(&unixsocket, "")},
       {"unixsocketperm", true, new OctalField(&unixsocketperm, 0777, 1, INT_MAX)},
+      {"logcleanerday", false, new IntField(&logcleanerday, -1, -1, INT_MAX)},
```

Comment:
ok ， i will do it

Reply:
how about renaming it to log-overdue-days ，becaue i find related var in the glog is named overdue_days_

Reply:
I prefer to use `retention` instead of overdue since it looks more clear than overdue. To see if other folks have thoughts on this. @PragmaTwice @torwig @caipengbo

Reply:
> I prefer to use retention instead of overdue since it looks more clear than overdue. To see if other folks have thoughts on this. 

I also think `retention` is better, which is a more general word.